src/main/java/frc/robot/subsystems/scoring/AimerIORoboRio.java

```diff
@@ -0,0 +1,252 @@
+    }
+
+    public void setStatorCurrentLimit(double limit) {
+        TalonFXConfigurator aimerLeftConfig = aimerLeft.getConfigurator();
+        aimerLeftConfig.apply(
+                new CurrentLimitsConfigs()
+                        .withStatorCurrentLimit(limit)
+                        .withStatorCurrentLimitEnable(true));
+
+        TalonFXConfigurator aimerRightConfig = aimerRight.getConfigurator();
+        aimerRightConfig.apply(
+                new CurrentLimitsConfigs()
+                        .withStatorCurrentLimit(limit)
+                        .withStatorCurrentLimitEnable(true));
+    }
+
+    @Override
+    public void setMotorDisabled(boolean disabled) {
+        motorDisabled = disabled;
+    }
+
+    @Override
+    public void updateInputs(AimerIOInputs inputs) {
+        appliedVolts = 0.0;
+
+        State trapezoidSetpoint =
+                profile.calculate(
+                        timer.get(),
+                        new State(initialAngle, initialVelocity),
+                        new State(goalAngleRad, 0));
+
+        double controlSetpoint =
+                MathUtil.clamp(trapezoidSetpoint.position, minAngleClamp, maxAngleClamp);
+        double velocitySetpoint = trapezoidSetpoint.velocity;
+
+        if (getEncoderPosition() == -1.75) {
+            motorDisabled = true;
+        }
+
+        if (override) {
+            appliedVolts = overrideVolts;
+        } else {
+            double controllerVolts = controller.calculate(getEncoderPosition(), controlSetpoint);
+            appliedVolts =
+                    feedforward.calculate(controlSetpoint, velocitySetpoint) + controllerVolts;
+        }
+
+        appliedVolts = MathUtil.clamp(appliedVolts, -12.0, 12.0);
+
+        Logger.recordOutput("Scoring/motorDisabled", motorDisabled);
+
+        inputs.aimGoalAngleRad = goalAngleRad;
+        inputs.aimProfileGoalAngleRad = controlSetpoint;
+        inputs.aimAngleRad = getEncoderPosition();
+
+        double currentTime = Utils.getCurrentTimeSeconds();
+        double diffTime = currentTime - lastTime;
+        lastTime = currentTime;
+
+        inputs.aimVelocityRadPerSec = (getEncoderPosition() - lastPosition) / diffTime;
+        velocity = (getEncoderPosition() - lastPosition) / diffTime;
+        lastPosition = getEncoderPosition();
+
+        inputs.aimVelocityErrorRadPerSec =
+                ((getEncoderPosition() - controlSetpoint) - lastError) / diffTime;
+        lastError = getEncoderPosition() - controlSetpoint;
+
+        inputs.aimAppliedVolts = appliedVolts;
+        inputs.aimStatorCurrentAmps = aimerRight.getStatorCurrent().getValueAsDouble();
+        inputs.aimSupplyCurrentAmps = aimerRight.getSupplyCurrent().getValueAsDouble();
+    }
+
+    @Override
+    public void applyOutputs(AimerIOInputs inputs) {
+        if (!motorDisabled || override) {
+            aimerRight.setVoltage(inputs.aimAppliedVolts);
+        } else {
+            aimerRight.setVoltage(0.0);
+        }
```

Comment:
I think we should move the output computations into this function, and then have this if/else for deciding what volts to apply. If it makes sense, we can have an AimerOutputs class that is autologged and represents the computed control targets.

Maybe we should rename `AimerIOInputs` to `AimerInputs` (and the same for similar other IO interfaces) because it currently is "Aimer Input Output Inputs" which doesn't actually make sense when you think about it. I'd be fine if we did that in a separate PR though since this one is big (please create a ticket for this)